### PR TITLE
[Refactoring]: Image Captioning Task

### PR DESCRIPTION
**Pull request by gitgoap**
## Tackling issue #6 Image Captioning Task

 I've moved `vanilla_ft.py`  scripts along with the `vanilla_paligemma_ft.ipynb` notebook into a new folder `image_captioning`.
 I have also added a README.md for the same.

@ariG23498

### CODE REVIEW

File: image_captioning/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Image Captioning with PaliGemma
+
+This project demonstrates fine-tuning the PaliGemma vision-language model for image captioning. 
+
+---
```
Comment (ariG23498) [inline image_captioning/README.md]: Can we remove the `---`?

**Review by ariG23498 — CHANGES_REQUESTED** [review]
Minor nit

File: image_captioning/README.md (hunk 1 above)
Comment (gitgoap) [reply image_captioning/README.md]: Sure

**Review by ariG23498 — APPROVED** [review]
LGTM!

**Pull request merged**